client/src/main/java/org/apache/uniffle/client/impl/ShuffleReadClientImpl.java

```diff
@@ -60,6 +60,8 @@ public class ShuffleReadClientImpl implements ShuffleReadClient {
   private AtomicLong crcCheckTime = new AtomicLong(0);
   private ClientReadHandler clientReadHandler;
   private final IdHelper idHelper;
+  private int fallbackTimes;
+  private int maxFallbackTimes = 3;
```

Comment:
Why should `maxFallbackTimes` be 3?

Reply:
Most jobs can run successfully without fallback mechanism, so it think 3 is enough. And it's difficult to pass parameters in the client. Should we add an new configration class?

Reply:
We usually use constructor to pass the config option which we need.